src/components/dialog/dialog.spec.js

```diff
@@ -818,7 +818,7 @@ describe('$mdDialog', function() {
 
       it('should not wrap raw content with md-dialog', inject(function($mdDialog, $rootScope) {
 
-        var template = '<div id="rawContent">Hello</div>';
+        var template = '<md-dialog id="rawContent">Hello</md-dialog>';
```

Comment:
The test was indeed passed erroneously. From a quick look at the docs, it's not clear what the user should pass as template if `atuoWrap` is false.

1. Could it be an HTML string that contains `<md-dialog>` ?
2. Could it be an HTML string containing a directive whose template contains `<md-dialog>` ?
3. Could it be any HTML string ?

(Based on the test I assume (3) isn't an option. If (2) is an option, then I would test for that here. If only (1) is possible, then I think the test above covers it, so this might be redundant.)

--
But, yeah, I meant that the docs should be more explicit (unless I missed that).